thirdparty/github.go

```diff
@@ -252,12 +255,34 @@ func getGithubClientRetry(token, caller string) *http.Client {
 	)
 }
 
+func GetInstallationToken(ctx context.Context, owner, repo string, opts *github.InstallationTokenOptions) (string, error) {
+	settings, err := evergreen.GetConfig()
+	if err != nil {
+		return "", errors.Wrap(err, "getting evergreen settings")
+	}
+	token, err := settings.CreateInstallationToken(ctx, owner, repo, opts)
+	if err != nil {
+		grip.Info(message.Fields{
+			"message": "creating token",
+			"ticket":  "EVG-19966",
+			"owner":   owner,
+			"repo":    repo,
+			"err":     err.Error(),
```

Comment:
It looks like the other debug statements are using grip.Debug, do we want to do that for this as well?